plugins/modules/packaging/os/homebrew.py

```diff
@@ -532,10 +541,13 @@ def _upgrade_all(self):
             self.changed = True
             self.message = 'Homebrew packages would be upgraded.'
             raise HomebrewException(self.message)
-        rc, out, err = self.module.run_command([
-            self.brew_path,
-            'upgrade',
-        ])
+        opts = (
+            [self.brew_path, 'upgrade']
+            + self.upgrade_options
+        )
+        cmd = [opt for opt in opts if opt]
```

Comment:
If you init upgrade_options as stated above you don't need this list comprehension anymore